conda.yaml

```diff
@@ -8,10 +8,10 @@ channels:
 dependencies:
   # Define conda-forge packages here -> https://anaconda.org/search
   # When available, prefer the conda-forge packages over pip as installations are more efficient.
-  - python=3.9.13               # https://pyreadiness.org/3.9/
-  - camelot-py=0.10.1
-  - pip=22.1.2                  # https://pip.pypa.io/en/stable/news/
+  - python=3.10.13               # https://pyreadiness.org/3.9/
+  - pip=23.3.2                  # https://pip.pypa.io/en/stable/news/
```

Comment:
Have we already jumped to these newer upgrades? By looking at the [template](https://github.com/robocorp/template-python-browser/blob/master/conda.yaml#L9-L11), it looks like we're still on the safe side with some lower versions for `python` and `pip`. Also `robocorp-truststore` is missing from these and it became the norm I think.

Reply:
Oh we are, but just not on that minor version yet.